### PR TITLE
Match idf surfaces

### PR DESCRIPTION
**Pull request by katsuya0719**
I modified the code related to [this issue](https://github.com/jamiebull1/geomeppy/issues/192).  
Also added the test case in which previous code doesn't pass and current code pass.

### CODE REVIEW

File: geomeppy/geom/intersect_match.py [hunk 1]
```diff
@@ -55,7 +55,8 @@ def match_idf_surfaces(idf):
                 set_unmatched_surface(surface, vector)
             matches = planes.get(-distance, {}).get(-vector, [])
             for s, m in product(surfaces, matches):
-                if almostequal(s.coords, reversed(m.coords)):
+                if almostequal(sorted(s.coords),sorted(m.coords)):
+                #if almostequal(s.coords, reversed(m.coords)):
```
Comment (jamiebull1) [inline geomeppy/geom/intersect_match.py:59]: Remove commented line please

File: tests/test_intersect_match.py [hunk 2]
```diff
@@ -399,6 +400,20 @@ def test_intersect_idf_surfaces(self, base_idf):
...
 
+    def test_match_idf_surfaces(self,stack_idf):
+        idf=stack_idf
+        starting = len(idf.idfobjects["BUILDINGSURFACE:DETAILED"])
+        idf.intersect_match()
+        ending = len(idf.idfobjects["BUILDINGSURFACE:DETAILED"])
+        assert starting == 12
+        assert ending == 14
+        #check outside boundary condition of roof
+        adjascent_cnt=0
```
Comment (jamiebull1) [inline tests/test_intersect_match.py:411]: Prefer to spell out in full (and fix the typo) `adjacency_count`.

File: tests/test_intersect_match.py [hunk 3]
```diff
@@ -399,6 +400,20 @@ def test_intersect_idf_surfaces(self, base_idf):
             obj = idf.getobject("BUILDINGSURFACE:DETAILED", name)
             assert obj
 
+    def test_match_idf_surfaces(self,stack_idf):
+        idf=stack_idf
+        starting = len(idf.idfobjects["BUILDINGSURFACE:DETAILED"])
```
Comment (jamiebull1) [inline tests/test_intersect_match.py:405]: I prefer more explicit naming of expected values, like this.

```
expected_surface_count = 12
assert len(idf.idfobjects["BUILDINGSURFACE:DETAILED"]) == expected_surface_count 
idf.intersect_match()
new_expected_surface_count = 14
assert len(idf.idfobjects["BUILDINGSURFACE:DETAILED"]) == new_expected_surface_count 
```

File: tests/conftest.py [hunk 4]
```diff
@@ -72,6 +73,16 @@ def ring_idf():
 
     return IDF(StringIO(ring_idf_txt))
 
+@pytest.fixture()
```
Comment (jamiebull1) [inline tests/conftest.py:76]: Please build the test case in code, rather than reading an IDF.

Comment (katsuya0719) [reply tests/conftest.py]: I simplified the test code to build the case in code.
